upup/pkg/fi/resources.go

```diff
@@ -119,6 +119,10 @@ type StringResource struct {
 	s string
 }
 
+func (r *StringResource) MarshalJSON() ([]byte, error) {
+	return json.Marshal(&r.s)
```

Comment:
I think this one is deep in the json marshalling machinery and I don't know what magic errors is uses....

I actually added this one just so the output would appear in the integration test output ... we're not actually using it per-se.